examples/common/sample_config.py

```diff
@@ -28,9 +28,18 @@
 class ActionWrapper(argparse.Action):
     def __init__(self, action):
         self._action = action
-        super().__init__(action.option_strings, action.dest, nargs=action.nargs, const=action.const,
-                         default=action.default, type=action.type, choices=action.choices, required=action.required,
-                         help=action.help, metavar=action.metavar)
+        super().__init__(
```

Comment:
I propose to move the formatting examples into a separate PR so as not to delay the merging of this PR.

Reply:
yeap, good idea.